src/index.js

```diff
@@ -42,7 +43,11 @@ async function createTestCafe (hostname, port1, port2) {
         getValidPort(port2)
     ]);
 
-    return new TestCafe(hostname, port1, port2);
+    var testcafe = new TestCafe(hostname, port1, port2);
+
+    exitHook(cb => testcafe.close().then(cb));
```

Comment:
> Test run stopping

Will this message be shown if I press `Ctrl+C` when tests are not running, e.g. when TestCafe is waiting for the remote worker to connect?